zerver/views/webhooks/updown.py

```diff
@@ -0,0 +1,79 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+import re
+from datetime import datetime, timedelta
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+
+SUBJECT_TEMPLATE = "Your service is {}"
```

Comment:
For this sort of uptime/downtime service notification, it's best if the subject is the same for the "up" and "down" notifications, so that they nicely thread together in Zulip's history.  So we should remove the status from the subject template, and replace it with some name/identifier for the particular service we're indicating the status of.  

For Updown, I think that means we want the subject to be simply event['check']['url'].  Does that reasoning make sense to you?  

And then maybe to make sure the up/down status is emphasized properly, maybe we should use bold (``bold``) for the word `up` or `down` in the body of the messages?